api/models/PrintingForm3D.js

```diff
@@ -34,7 +34,7 @@ const PrintingForm3DSchema = new Schema(
 
     // unique id for query
     // id = email of the user who requested
-    id: {
+    email: {
```

Comment:
you will need to refactor the `3DPrintingForm` tests with this change. for example see [here](https://github.com/SCE-Development/Core-v4/blob/9127fd37323555dffde5644f9f19934d2fdbc9ed/test/3DPrintingForm.js#L72)